geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/set/SScanExecutor.java

```diff
@@ -66,12 +67,13 @@ public RedisResponse executeCommand(Command command, ExecutionHandlerContext con
 
     RedisKey key = command.getKey();
 
-    if (!getDataRegion(context).containsKey(key)) {
+    RedisData value = context.getRegionProvider().getRedisData(key, null);
```

Comment:
this is another place that could be changed to "context.getRegionProvider().getRedisData(key);
      if (value.isNull()"

Reply:
Fixed